tee-worker/client-api/.prettierrc

```diff
@@ -1,5 +1,6 @@
 {
-	"printWidth": 140,
+  "printWidth": 140,
   "bracketSpacing": true,
-  "tabWidth": 4
+  "tabWidth": 4,
+  "singleQuote": false
```

Comment:
opportunistic change: I noticed that there is some mix between single and double quotes. The former seems predominates. Made it explicit in prettier.